README.md

```diff
@@ -58,6 +58,12 @@ Any changes to the first configuration must also be reported in the latter.
 For OpenShift-related configuration in the chart visit [here](https://github.com/bitnami/charts/blob/main/bitnami/postgresql/README.md#differences-between-bitnami-postgresql-image-and-docker-official-image).
 
 ## Installation
+**Note: ArgoCD and workflow namespace**
+If you manually the created the workflow namespaces (e.g., `sonatflow-infra`), run this command to add the required label that allows
```

Comment:
pls note that with existing deployment, any namespace can be used to run the workflows in.

Reply:
Yes, but the same README has `Deploying PostgreSQL reference implementation` that uses `sonataflow-infra` with no further details on how to customize it, so I followed the same (weak) approach